Sources/NIOTransportServices/NIOTSChannelOptions.swift

```diff
@@ -21,6 +22,19 @@ public struct NIOTSChannelOptions {
     public static let waitForActivity = NIOTSChannelOptions.Types.NIOTSWaitForActivityOption()
 
     public static let enablePeerToPeer = NIOTSChannelOptions.Types.NIOTSEnablePeerToPeerOption()
+    
+    public static let currentPath = Types.NIOTSCurrentPathOption()
+
+    @available(OSX 10.15, iOS 12.0, tvOS 12.0, watchOS 6.0, *)
+    public static let metadata = { (definition: NWProtocolDefinition) -> Types.NIOTSMetadataOption in
+        .init(definition: definition)
+    }
+
+    @available(OSX 10.15, iOS 12.0, tvOS 12.0, watchOS 6.0, *)
```

Comment:
I think this guard (and associated guards) are wrong. Per [the documentation](https://developer.apple.com/documentation/network/nwconnection/3200452-requestestablishmentreport), the establishment report was added in macOS 10.15, iOS 13, tvOS 13, and watchOS 6.

Reply:
The same applies to the data transfer report.